packages/react/src/utils.ts

```diff
@@ -482,6 +482,312 @@ export const sortByProperty = <T>(arr: T[], property: keyof T, order: SortOrder
   });
 };
 
+export const formatShortDateString = (date: Date) => {
+  if (!date) return date;
+
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, "0");
+  const day = String(date.getDate()).padStart(2, "0");
+
+  return `${year}-${month}-${day}`;
+};
+
+export const formatLongDateString = (date: Date) => {
+  if (!date) return date;
+
+  return date.toLocaleString("en-US", { year: "numeric", month: "short", day: "numeric" });
+};
+
+export const formatLongDateTimeString = (date: Date) => {
+  if (!date) return date;
+
+  return date
+    .toLocaleString("en-US", {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+      hour: "numeric",
+      minute: "2-digit",
+      hour12: true,
+    })
+    .replace(/,(?=[^,]*$)/, "");
```

Comment:
what's this for?

Reply:
That regex replaces the last comma in the returned date string with nothing.

The table used `date-fns`'s `format` to formatted the string like so:
`June 19, 2024 12:00 AM`
Without the `.replace`, the built-in formatter formats the string like so:
`June 19, 2024, 12:00 AM`

This was causing tests to fail, so I removed the last comma.